episodes/09-integrations.md

```diff
@@ -20,19 +21,90 @@ exercises: 0
 - Token
-
-If you are setting up an integration to your ILS, it is recommended to touch base with those who have already done this. For example, for Alma, you can reach out to the Alma listserv to get guidance.
-
-For the OCLC API integration, you will need to request an API key with OCLC. You will need the Registry API, Metadata API, and Search API. It is recommended to consult OCLC's help page on [setting up MarcEdit OCLC integration](https://help.oclc.org/Librarian_Toolbox/OCLC_APIs/Troubleshooting/How_do_I_set_up_MarcEdit_OCLC_Integration%3F?sl=en) as well as Terry Reese's information from his blog.
-
-For the Z39.50 integration, you will need to enter your username and password in the settings. For OCLC, this is your Connexion number and password. For the Z39.50 search for OCLC, there are options for the types of searches performed and translating records to UTF-8.
+- API Key
+
+### Integration with an ILS
+At this time, MarcEdit supports integrations with the ILS Alma and Koha. The ILS integration will require you to request an API key from your systems person, your ILS vendor, or your hosting vendor. You can also ask for instructions on how to set up the integration. For example, for Alma, you can reach out to the Alma listserv to get guidance or to the Alma Tech Blog on [integrating MarcEdit with Alma](https://developers.exlibrisgroup.com/blog/configure-marcedit-7-to-work-with-alma-updated-for-marcedit-7-5-116-12-17-2021/). For Koha, this [article](https://bywatersolutions.com/education/koha-tutorial-exporting-records-marcedit) from ByWater Solutions offers guidance on how to edit records in MarcEdit using the Koha MarcEdit integration.
+
+### Integration with OCLC's API
+For the OCLC API integration, you need to [request an API key and check if your institution is eligible](https://www.oclc.org/developer/support/eligibility.en.html) for this service. If your institution is eligible, then you can request a key through [OCLC's developer network](tps://www.oclc.org/developer/develop/authentication/how-to-request-a-wskey.en.html). You can also go to Preferences from the main menu in MarcEdit, click on OCLC API Integration and then click on the "Request an OCLC Key". It is recommended to consult OCLC's help page on [setting up MarcEdit OCLC integration]([https://help.oclc.org/Librarian_Toolbox/OCLC_APIs/Troubleshooting/How_do_I_set_up_MarcEdit_OCLC_Integration%3F?sl=en](https://help.oclc.org/Librarian_Toolbox/OCLC_APIs/Troubleshooting/How_do_I_set_up_MarcEdit_OCLC_Integration%3F)) as well as Terry Reese's information from his blog.
+
+### Integration with a Z39.50 Database
+Many systems rely on the Z39.50 protocol. It could be the case that your current ILS relies on it for Interlibrary loan. For this type of integration, you need the following information below. You can find this information at the Library of Congress [Gateway to Library Catalogs](https://www.loc.gov/z3950/) of [zbrary](https://www.z-brary.com/).
```

Comment:
```suggestion
Many systems rely on the Z39.50 protocol to search and exchange data. It could be the case that your current ILS relies on it for Interlibrary loan. For this type of integration, you need the following information below. You can find this information at the Library of Congress [Gateway to Library Catalogs](https://www.loc.gov/z3950/) of [zbrary](https://www.z-brary.com/).
```